pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -219,13 +219,24 @@ func (p *PoolManager) IsKubevirtEnabled() bool {
 	return p.isKubevirt
 }
 
-func isRelatedToKubevirt(pod *corev1.Pod) bool {
+func (p *PoolManager) isRelatedToKubevirt(pod *corev1.Pod) bool {
 	if pod.ObjectMeta.OwnerReferences == nil {
 		return false
 	}
 
 	for _, ref := range pod.OwnerReferences {
 		if ref.Kind == kubevirt.VirtualMachineInstanceGroupVersionKind.Kind {
+			requestUrl := fmt.Sprintf("apis/kubevirt.io/v1alpha3/namespaces/%s/virtualmachines/%s", pod.Namespace, ref.Name)
+			log.V(1).Info("test", "requestURI", requestUrl)
+			result := p.kubeClient.ExtensionsV1beta1().RESTClient().Get().RequestURI(requestUrl).Do()
```

Comment:
why do you use RESTClient instead of kubeClient VirtualMachines?

Reply:
because to import the kubecli from kubevirt I need to add multiple packages like `gorilla/websocket` and others and I don't want to do it just to check if the object exist.

I import only the api package from kubevirt.

Reply:
Sounds good